src/cgame/cg_tutorial.cpp

```diff
@@ -545,15 +549,13 @@ static void CG_SpectatorText( char *text, playerState_t *ps )
 		else if ( cgs.clientinfo[ cg.clientNum ].team == TEAM_NONE )
 		{
 			Q_strcat( text, MAX_TUTORIAL_TEXT,
-			          va( _( "Press %s to return to free spectator mode\n" ),
-			              CG_KeyNameForCommand( "+attack3" ) ) );
-		}
-		else
-		{
-			Q_strcat( text, MAX_TUTORIAL_TEXT,
-			          va( _( "Press %s to stop following\n" ),
+			          va( _( "Press %s to return to first-person spectator mode\n" ),
 			              CG_KeyNameForCommand( "+attack3" ) ) );
 		}
+
+		Q_strcat( text, MAX_TUTORIAL_TEXT,
+		          va( _( "Press %s to follow the next player\n" ),
+		              CG_KeyNameForCommand( "weapnext" ) ) );
 	}
 	else
 	{
```

Comment:
Looks like this case needs to be updated.

Reply:
When spectating, it's `attack1` to join a team anytime, `attack3` to start to follow a player, then `attack3` to cycle between 1st person view or chase, `weapnext`/`weapprev` to switch player to follow, and `attack2` to cancel following and return to free fly spectating.